src/AspNet.Security.OAuth.Trovo/AspNet.Security.OAuth.Trovo.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
Could you also add this please?

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/f51b62b4b63406ef04198bb780084c2fedabc6d7/src/AspNet.Security.OAuth.Twitter/AspNet.Security.OAuth.Twitter.csproj#L4